savant/utils/memory_repr_pytorch.py

```diff
@@ -0,0 +1,40 @@
+import cv2
+import torch
+
+from savant.utils.memory_repr import OpenCVGpuMatCudaArrayInterface, \
+    _numpy_type_to_opencv_mat_type
+
+
+__all__ = [
+    'opencv_gpu_mat_as_pytorch',
+    'pytorch_tensor_as_opencv_gpu_mat',
+]
+
+
+def opencv_gpu_mat_as_pytorch(gpu_mat: cv2.cuda.GpuMat) -> torch.Tensor:
+    """Returns PyTorch tensor for specified OpenCV GpuMat."""
```

Comment:
Here we can add a note about the format HWC/CWH

Reply:
Added